### ISSUE TITLE
Feature request for onenote. Put notebook content in a folder named after the notebook.

### ISSUE THREAD

**Issue by legolasdimir** [obsidianmd/obsidian-importer#145]
When creating the folders from an import, the plugin currently makes a folder named the same as the section that the pages are in. 

For organization purposes when you are trying to import those notes into your vault, it would make more sense at least to me, to have the content be organized one more time in a folder named after the notebook it came out of, then the section, with the pages in that section. Much like the content is organized within OneNote

**Comment by kepano** [comment]
Seems like a duplicate of #165

**Comment by legolasdimir** [comment]
@kepano I would agree. the other issue isn't super clear. At least I did not think it was.

**Comment by p3rid0t** [comment]
The current importer makes a structure like this:
```
- Output folder (or vault root)
  - Section group
      - Section
          - Page 1
      - Section 2
          - Page 1
```

From my understanding, you are expecting the importer to create another folder for notebooks, like this?
```
- Output folder (or vault root)
   - Notebook
     - Section group
         - Section
            - Page 1
         - Section 2
            - Page 1
```

**Comment by legolasdimir** [comment]
yes. so everything is grouped by notebook. If it's not it breaks whatever organization structure was in the notes already.

**Comment by legolasdimir** [comment]
it also still doesn't actually import my notes. folders are empty.

**Comment by p3rid0t** [comment]
This seems like a good idea - for simplicity, notebook folders will only get created when you select sections from multiple notebooks.

Regarding your issue with empty folders - could you please check the Obsidian console accessible by pressing `Ctrl` + `Shift` + `I` (or `Cmd` + `Opt` + `I`) and send any errors that show up?
